styles.css

```diff
@@ -3,9 +3,28 @@
 }
 
 body {
-  background-color: midnightblue;
+  background-color: seagreen;
 }
 
 .race-car {
   height: 120px;
 }
+
+.east {
+  transition-duration: 0.25s;
+}
+
+.north {
+  transform: rotate(-90deg);
+  transition-duration: 0.25s;
+}
+
+.south {
+  transform: rotate(90deg);
+  transition-duration: 0.25s;
+}
+
+.west {
+  transform: rotate(180deg);
+  transition-duration: 0.25s;
```

Comment:
Instead of applying `transition-duration` in each rule here, you should only apply it once in the rule for `.race-car`